src/command/build.rs

```diff
@@ -38,6 +38,8 @@ pub enum BuildMode {
     /// Don't install tools like `wasm-bindgen`, just use the global
     /// environment's existing versions to do builds.
     Noinstall,
+    /// Skip the rustc version check
```

Comment:
I can imagine there are cases where I want to use the existing wasm-bindgen and whatever rustc I have, and just do dang it.

By making `force` a build mode, are we making this mutually exclusive with `no-install`?

Reply:
yeah so my thought is "dont check my stuff" = "force"
don't friggin install *anything* = "no-install"
i guess if you want both.. we don't let that happen at the moment, we'll see if someone asks for it?

Reply:
in my mind, no install is for docker users. i think docker users still want checks.